store/postgres/migrations/2019-05-15-215022_migrate_entities/up.sql

```diff
@@ -0,0 +1,173 @@
+create type deployment_schema_version as enum ('public', 'split');
+create type deployment_schema_state as enum ('ready', 'tables');
+
+alter table deployment_schemas
+  add column version   deployment_schema_version,
+  add column migrating bool not null default false,
+  add column state     deployment_schema_state not null default 'ready';
+
+update deployment_schemas
+   set version='split';
+
+alter table deployment_schemas
+  alter column version set not null;
+
+insert into deployment_schemas(subgraph, version)
+select id, 'public'::deployment_schema_version
+  from subgraphs.entities e
+ where entity='SubgraphDeployment'
+   and not exists (select 1 from deployment_schemas d
+                           where d.subgraph=e.id);
```

Comment:
I'm deducting from previous migrations and the state of my local DB that the only column we had in this table was the subgraph of subgraphs, and its `version` is null. Does this `insert` assume those things? If so could that be added in a comment?

Reply:
Up to this point, the only entries in `deployment_schemas` were the subgraph of subgraphs, and any subgraphs that were deployed after commit `d2248402`. With the changes in this PR, we need entries for all subgraphs in `deployment_schemas` (I also added comments to the migration to that effect)